app/models/manageiq/providers/base_manager/metrics_capture.rb

```diff
@@ -199,23 +236,26 @@ def perf_capture_queue_target(target, interval_name, start_time:, end_time:, tas
       :zone        => my_zone,
       :state       => ['ready', 'dequeue'],
     }
-    queue_item[:args] = [start_time, end_time] if start_time
+    queue_item[:args] = [start_time, end_time, target_ids, rollup] if start_time || target_ids.present? || rollup
```

Comment:
Not for this PR, but is there any downside to passing all these options all the time even if they are nil?

Reply:
I would have preferred to skip nulls / use default parameters.
It ended up having quite a few variations for these queries.